topcoffea/data/TauSF/README.md

```diff
@@ -0,0 +1,8 @@
+# JSONs containing corrections and systematics
+
+Apart from the files starting with `TauFakeSF_*`, the documentation and the source of the files contained here is listed TAU POG GH repo: https://github.com/cms-tau-pog/TauIDSFs.git. When necessary, the central files where translated in the legacy JSON format used for TOP-22-006.
+
+The SFs and systematic uncertainties contained in `TauFakeSF_*` represent the corrections to the estimate of the jet faking hadronic taus at a fixed WP of the `DeepTau` discriminator.
+The procedure was presented to TAU POG: https://indico.cern.ch/event/1326438/#preview:4727537
+
+The actual code used to produce the latter SFs will be committed ASAP
```

Comment:
I think it might make sense to remove this from the readme and just add an issue to the repo.

Reply:
Well, maybe we can keep this as first reference, and update it later on

Reply:
@anpicci I think this (and my other comment on the conda version) were all the remains.